### PR TITLE
Adding Home -> Participants Folder

### LINKED ISSUE #9: Home Page - Participants
https://www.figma.com/file/KSkquGGDY2MQGrXsTFaAaL/Untitled?node-id=7%3A3

### PR DESCRIPTION
**Pull request by AAYUSHAIRAN**
Closes #9

### CODE REVIEW

**Comment by AAYUSHAIRAN** [comment]
@parth-paradkar  don't merge,
yet to add a responsive part 
Will add by tomorrow

File: src/components/Home/components/participants/participants.js [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+          fontStyle: "normal",
+          fontWeight: "600",
+          fontSize: "60px",
+
+          textAlign: "center",
+          color: "#FFFFFF",
+          backgroundColor: "black",
+          margin: "0px",
+          letterSpacing: "4px",
+        }}
```
Comment (parth-paradkar) [inline src/components/Home/components/participants/participants.js]: Inline style

File: src/components/Home/components/participants/participants.js [hunk 2]
```diff
@@ -1,5 +1,72 @@
...
+      >
+        Participants
+      </h1>
+      <div
+        style={{
+          margin: "0px 118px",
+          height: "171px",
+          borderBottom: "6px solid white",
+        }}
```
Comment (parth-paradkar) [inline src/components/Home/components/participants/participants.js]: Inline styles

**Review by parth-paradkar — CHANGES_REQUESTED** [review]
Shift inline styles to CSS file

**Review by parth-paradkar — APPROVED** [review]

**Pull request merged**